int16 left_distance
int16 right_distance
